src/js/game/hud/parts/mass_selector.js

```diff
@@ -41,6 +41,7 @@ export class HUDMassSelector extends BaseHUDPart {
         this.deletionMarker = Loader.getSprite("sprites/misc/deletion_marker.png");
 
         this.currentSelectionStart = null;
```

Comment:
Could you change it so it only uses `currentSelectionStartWorld`? No need for two variables

Reply:
fixed